kivy/core/window/window_sdl2.py

```diff
@@ -311,11 +311,15 @@ def create_window(self, *largs):
             # setup window
             w, h = self.system_size
             resizable = Config.getboolean('graphics', 'resizable')
+            show_taskbar_icon = Config.getboolean(
+                'graphics',
+                'show_taskbar_icon'
+            )
```

Comment:
What about moving the logic into `setup_window` method  (Like we're doing for `multisamples = Config.getint('graphics', 'multisamples')`) ?

This will allow us to not add an additional argument to the `setup_window` method.